admin/src/main/scala/za/co/absa/spline/arango/foxx/FoxxManagerImpl.scala

```diff
@@ -31,23 +27,10 @@ class FoxxManagerImpl(restClient: RESTClient)(implicit ec: ExecutionContext)
   extends FoxxManager
     with Logging {
 
-  override def install(mountPrefix: String, assets: Array[(String, String)]): Future[Unit] = {
+  override def install(mountPrefix: String, content: Array[Byte]): Future[Unit] = {
```

Comment:
It took me a while to realize why the zipping is not needed anymore (using `maven-assembly-plugin` with `<format>zip</format>` defined in new modules' `assembly.xml`).

Is there an obvious upside for this approach? I know, it is declarative, but it seems that the configuration to achieve this is quite big.

Reply:
Well, it was not done on purpose, I just made use of the fact the module is now packed as zip anyway. The intention was to split a TS based Foxx code from Scala based application level code running on a different tier. And since TS is not Java, the most logical distribution format is ZIP. And the Foxx package format is ZIP. So why would we pack-unpack-pack then?

Reply:
😆 oh, I see